README.md

```diff
@@ -196,6 +196,19 @@ quicklink({
 });
 ```
 
+You may also wish to ignore prefetches to URLs which contain a URL fragment (e.g. `index.html#top`). This can be useful if you (1) are using anchors to headings in a page or (2) have URL fragments setup for a single-page application, and which to avoid firing prefetches for similar URLs.
+
+Using `ignores` this can be achieved as follows:
+
+```js
+quicklink({
+    ignores: [
+        uri => uri.includes('#')
+        // or /#(.+)/ if you prefer RegExp
```

Comment:
And also element matching:

```js
// or
(uri, elem) => !!elem.hash
``

Reply:
Great idea. Adding in the next commit. Cheers!